### PR TITLE
Edit some LoginPage

### PR DESCRIPTION
**Pull request by TuzkiYc**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -12,15 +12,17 @@
   },
   "dependencies": {
     "@emotion/react": "^11.11.4",
-    "@emotion/styled": "^11.11.0",
+    "@emotion/styled": "^11.11.5",
```
Comment (KevinHChen) [inline frontend/package.json:15]: remove

File: frontend/package.json [hunk 2]
```diff
@@ -12,15 +12,17 @@
...
     "@mui/icons-material": "^5.15.14",
-    "@mui/material": "^5.15.13",
+    "@mui/material": "^5.15.16",
+    "@mui/styled-engine-sc": "^6.0.0-alpha.18",
```
Comment (KevinHChen) [inline frontend/package.json:18]: remove

File: frontend/package.json [hunk 3]
```diff
@@ -12,15 +12,17 @@
...
     "@mui/system": "^5.15.14",
     "@react-oauth/google": "^0.12.1",
     "axios": "^1.6.8",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.22.3",
+    "styled-components": "^6.1.9",
```
Comment (KevinHChen) [inline frontend/package.json:25]: remove it

File: frontend/src/components/AuthPage/AuthPageProvider.jsx [hunk 4]
```diff
@@ -29,46 +14,77 @@ const AuthPageProvider = () => {
...
+                required
+                fullWidth
+                name="password"
+                label="Password"
+                type="password"
+                id="password"
+                autoComplete="current-password"
+              />
+              <Button
+                type="submit"
```
Comment (KevinHChen) [inline frontend/src/components/AuthPage/AuthPageProvider.jsx:68]: set it as `disabled`

File: frontend/src/components/AuthPage/AuthPageProvider.jsx [hunk 5]
```diff
@@ -29,46 +14,77 @@ const AuthPageProvider = () => {
...
+            <Box component="form" noValidate sx={{ mt: 1 }}>
+              <TextField
+                margin="normal"
+                required
+                fullWidth
+                id="username"
+                label="Email"
+                name="username"
+                autoComplete="username"
+                autoFocus
```
Comment (KevinHChen) [inline frontend/src/components/AuthPage/AuthPageProvider.jsx:55]: set it as `disabled`

File: frontend/src/components/AuthPage/AuthPageProvider.jsx [hunk 6]
```diff
@@ -29,46 +14,77 @@ const AuthPageProvider = () => {
...
+              />
+              <TextField
+                margin="normal"
```
Comment (KevinHChen) [inline frontend/src/components/AuthPage/AuthPageProvider.jsx:58]: set element as `disabled`

**Review by KevinHChen — CHANGES_REQUESTED** [review]
1. Some packages are not required so shall not be committed
2. Disable the username/password login, because you are passing it a form, which will expose the username and password onto the address bar, which is not a secure practise.

**Comment by MarkZhuVUW** [comment]
> 1. Some packages are not required so shall not be committed
> 2. Disable the username/password login, because you are passing it a form, which will expose the username and password onto the address bar, which is not a secure practise.
>    image

we were never going to do username & password login. It has been decided at project prososal

**Comment by MarkZhuVUW** [comment]
there is no time for username & password login.

We already have google sso

We have decided that we will do google sso only at affinity estimate stage (probably even at project proposal stage)

can we just remove the text boxes?

just include some text description above the google sso login button should be fine

**Pull request closed without merging**